openmetadata-ui/src/main/resources/ui/src/styles/components/tabs.less

```diff
@@ -31,6 +31,6 @@
 .custom-tab-spacing {
   .ant-tabs-nav {
     margin: 0 !important;
-    padding: 0 24px;
+    padding: 0 16px;
```

Comment:
don't override global styles, is you need to change something do at your component level.

Reply:
Updated